Sources/Modlog.php

```diff
@@ -163,6 +163,7 @@ function ViewModlog()
 	// This is all the information required for a watched user listing.
 	$listOptions = array(
 		'id' => 'moderation_log_list',
+		'title' => $context['log_type'] === 3 ? $txt['admin_log'] : $txt['moderation_log'],
```

Comment:
Since we historically haven't perform strict checks and due to the volatile nature of global vars, please use == insead of ===